pkg/state/common_test.go

```diff
@@ -25,18 +25,28 @@ import (
 
 	mellanoxv1alpha1 "github.com/Mellanox/network-operator/api/v1alpha1"
 	clustertype_mocks "github.com/Mellanox/network-operator/pkg/clustertype/mocks"
+	"github.com/Mellanox/network-operator/pkg/consts"
 	"github.com/Mellanox/network-operator/pkg/state"
 	"github.com/Mellanox/network-operator/pkg/staticconfig"
 	staticconfig_mocks "github.com/Mellanox/network-operator/pkg/staticconfig/mocks"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/resource"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	defaultTestRepository = "myRepo"
+	defaultTestImage      = "myImage"
+	defaultTestVersion    = "myVersion"
+)
+
```

Comment:
In https://github.com/Mellanox/network-operator/pull/996 I am moving the consts to the test scope. Otherwise, the complete `state_test` package will have these constants. It is pretty confusing if we define consts here and we can use them anyway. We should define and pass them explicitly, WDYT

Reply:
I have those three global because many tests need to define an image in `.spec`.

Reply:
Ah, it's in the `common_test.go`, overlooked that. Thought I was in the state test